app/index.html

```diff
@@ -1,18 +1,43 @@
 <!DOCTYPE html>
-<html lang="">
+<html lang="en">
   <head>
-    <meta charset="utf-8" />
-    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
-    <meta name="viewport" content="width=device-width,initial-scale=1.0" />
-    <link rel="icon" href="/favicon.ico" />
+    <title>dGRANTS</title>
+    <meta http-equiv="Content-Type" content="text/html; charset=UTF-8" />
+    <meta
+      name="viewport"
+      content="width=device-width, initial-scale=1, maximum-scale=1.0, user-scalable=no"
+    />
+    <meta name="robots" content="index,follow" />
+    <meta name="audience" content="All" />
+    <meta name="revisit-after" content="7 days" />
+    <meta name="description" content="" />
+    <meta name="keywords" content="" />
+
+    <meta property="og:title" content="dGRANTS – xxx" />
+    <meta property="og:description" content="xxx" />
+    <meta property="og:image" content="dgrants_social_preview.png" />
+    <meta property="og:url" content="https://xxx" />
+    <meta name="twitter:card" content="summary" />
+    <meta name="twitter:site" content="@gitcoin" />
+    <meta name="twitter:title" content="dGRANTS – xxx" />
+    <meta name="twitter:description" content="xxx" />
+    <meta name="twitter:image" content="dgrants_social_preview.png" />
```

Comment:
> REMINDER : In index.html ( Line 13-24 )
> 
> - have to put real content in for keywords, cards, texts
> - need to link to an absolute https:// url for the dgrants_social_preview.png
> - need to edit the real https:// address of our project

@phutchins Do you have any idea of what the official URL will be yet? Also @scco do you think we can get the real data for this placeholders as part of this PR, or should it be a different PR with an issue to track it?